noxfile.py

```diff
@@ -226,16 +226,27 @@ def pull_artifacts(session):
 
 @nox.session
 def install(session):
+    from pathlib import Path
+
+    tmp_lamindb_path = Path("tmp_lamindb")
+    tmp_lamindb_path.mkdir(parents=True, exist_ok=True)
+
     run(
         session,
-        "pip install --no-deps git+https://github.com/laminlabs/lnschema-core git+https://github.com/laminlabs/bionty git+https://github.com/laminlabs/lamindb-setup git+https://github.com/laminlabs/wetlab git+https://github.com/laminlabs/clinicore git+https://github.com/laminlabs/cellregistry git+https://github.com/laminlabs/ourprojects",
+        f"git clone -b main --depth 1 --recursive --shallow-submodules https://github.com/laminlabs/lamindb {str(tmp_lamindb_path)}",
```

Comment:
Why not use the `install_lamindb()` function from laminci we use everywhere else? 🤔